server/jobs/index.js

```diff
@@ -0,0 +1,20 @@
+import { scheduleJob } from 'node-schedule'
+
+import { updateAllScores } from '../helpers/index'
+
+// Ensures job only runs during Katathon
+export const newKatathonJob = (timestamp, katathonId) => {
+  const start = new Date(timestamp)
+  const end = new Date(timestamp)
+  end.setHours(end.getHours() + 24)
+  scheduleJob(
+    {
+      start,
+      end,
+      rule: '*/10 * * * *'
+    },
+    () => {
+      updateAllScores(katathonId)
+    }
+  )
+}
```

Comment:
Really like this functionality. I have never seen that before.

Reply:
Yeah. Alex and I had to research how to do this